### PR TITLE
SCTE35-37 : updated readme

### PR DESCRIPTION
**Pull request by triple0t**
Updated Readme, added information for JavaScript Specific platform

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,39 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 [iOS Specific](https://github.com/realeyes-media/scte35-swift)
 
+[Javascript Specific](https://github.com/realeyes-media/scte35-js)
+
 ### How to Include on Your Platform
 
-TODO
+
+**Javascript**
```
Comment (mostafashaaban5) [inline README.md:27]: @triple0t Looks pretty good but can you clarify more regarding cloning the project first then where to find the js output folder ?

Comment (triple0t) [reply README.md:27]: @mostafashaaban5 thanks. I have added the Information to the js specific project

Should I replicate that here?

Comment (triple0t) [reply README.md:27]: @mostafashaaban5 I have added the Information to the js specific project.

Should I replicate that here?

Comment (mostafashaaban5) [reply README.md:27]: I think would be better

Comment (triple0t) [reply README.md:27]: Alright

**Pull request merged**